Formula/liblwgeom.rb

```diff
@@ -35,11 +25,13 @@ class Liblwgeom < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
   depends_on "gpp" => :build
 
   depends_on "proj"
   depends_on "geos"
   depends_on "json-c"
+  depends_on "libxml2" => :recommended
```

Comment:
Make that mandatory.

Reply:
macOS provides libxml2 so it's weird we'd need to add this

Reply:
OS provided libxml2 should be current enough so I've removed the depend for the time being. Can easily add it back if that is the preferred approach.